Breadcrumbs.sublime-settings

```diff
@@ -1,5 +1,7 @@
 {
-"breadcrumb_length_limit": 100,
-"total_breadcrumbs_length_limit": 200,
-"breadcrumbs_separator": " ■ "
+  "breadcrumb_regex": "^([{}\\(\\)\\[\\]\\s])*\\s*(?P<name>[^{\\(\\[]*[^:{\\(\\[\\s])",
```

Comment:
Oh, sorry... the correct answer is b :)

> (Do you know that in Python you can declare a string as "raw" using r instead of u, which lets you replace double slashes with single slashes?)

Well, I just Googled what u does earlier this evening and didn't think to find out other options. (Apparently the u is only necessary for ST2 which runs Python < 3). That might help.